### ISSUE TITLE
Describe what actionSanitizer / stateSanitizer do.

### ISSUE THREAD

**Issue by MNF** [zalmoxisus/redux-devtools-extension#661]
The documentation https://github.com/zalmoxisus/redux-devtools-extension/blob/master/docs/API/Arguments.md#actionsanitizer--statesanitizer said that actionsanitizer/statesanitizer are functions that return  object back, but doesn’t describe, what they do with objects.  
Clear description what they do and when they should be used will be useful.

**Comment by Marcidius** [comment]
Yes please!  I'm having significant issues using DevTools due to an oversized state [as described here.](https://github.com/zalmoxisus/redux-devtools-extension/issues/619)  Documentation elsewhere on how/when to implement these is severely lacking.

**Comment by SKEPDIMI** [comment]
Bump

**Comment by MRKrog** [comment]
Currently trying to learn about actionsanitizer/statesanitizer.

From I have gathered, with statesanitizer you can omit state from being displayed in your dev tools. For my case I have an application which is returning over 30k rows from an api call, which was causing way too much strain on my redux dev tools. By omitting this particular state from my dev tools display, my application is running much smoother.

I may be completely wrong in my implementation or use case of my situation, Let me know if anyone else has encountered a similar issue. As well as a use case for actionsanitizer.

Per example my composeEnhancer:_

const composeEnhancers = composeWithDevTools({
  stateSanitizer: function (state) {
    return {
      ...state,
      products: undefined,
    };
  },
});

in the example on the docs it would be like so:
 stateSanitizer: (state) => state.products ? { ...state, products: undefined } : state // (es6 same as above)
